packages/block-editor/src/components/global-styles/style.scss

```
@@ -75,3 +75,28 @@
 	/*rtl:ignore*/
 	direction: ltr;
 }
+
+.block-editor-panel-duotone-settings__dropdown {
+	position: relative;
+}
+
+.block-editor-panel-duotone-settings__reset {
+	position: absolute;
+	right: 0;
+	top: $grid-unit;
+	margin: auto $grid-unit auto;
+	opacity: 0;
+	transition: opacity 0.1s ease-in-out;
+	@include reduce-motion("transition");
```

```suggestion
	@media not ( prefers-reduced-motion ) {
		transition: opacity 0.1s ease-in-out;
	}
```

See https://github.com/WordPress/gutenberg/issues/68282